src/vdaf.rs

```diff
@@ -248,6 +248,21 @@ pub trait Aggregator<const VERIFY_KEY_SIZE: usize, const NONCE_SIZE: usize>: Vda
         agg_param: &Self::AggregationParam,
         output_shares: M,
     ) -> Result<Self::AggregateShare, VdafError>;
+
+    #[cfg(feature = "experimental")]
+    /// This is special dpsa-project functionality.
+    /// In order to implement noising for differential privacy,
+    /// We define a custom postprocessing function for vdafs,
+    /// such that each aggregator can call it on its aggregate shares.
+    ///
+    /// The default implementation is the identity function.
+    fn postprocess(
+        &self,
+        _agg_param: &Self::AggregationParam,
```

Comment:
Why does this take `Self::AggregationParam`? It doesn't look like it's used in the `Prio3` implementations. I agree with Chris' point that we should anticipate DP mechanisms that want to tune parameters based on the number of contributions. I think we should define an associated type on `trait vdaf::Aggregator` called `DifferentialPrivacyParam` and replace this with:

```suggestion
        _agg_param: &Self::DifferentialPrivacyParam,
```

If you did that, then you could use `flp::types::fixedpoint_l2::PrivacyParameterType` (or rather, whatever you rename that type to) as `Prio3FixedPointBoundedL2VecSum`'s `DifferentiaPrivacyParam`.

I think the important thing here is moving the DP params from `new` and into the `Vdaf::Aggregator` trait. It doesn't make sense to require clients to express DP parameters when they shard inputs, and the point of doing central/server DP is that clients don't have to be aware of it.

Reply:
Yes, this seems like a very sensible thing to do!

Reply:
For posterity: I argued in favor of keeping the aggregation parameter argument while bringing in the DP mechanism/parameters in a separate argument, over on [#607](https://github.com/divviup/libprio-rs/pull/607/files#r1232593896)